src/Fixer/PhpTag/BlankLineAfterOpeningTagFixer.php

```diff
@@ -86,6 +81,7 @@ public function getPriority()
      */
     public function isCandidate(Tokens $tokens)
     {
-        return $tokens->isTokenKindFound(T_OPEN_TAG);
+        // ignore files with short open tag and ignore non-monolithic files
+        return $tokens[0]->isGivenKind(T_OPEN_TAG) && $tokens->isMonolithicPhp();
```

Comment:
from public interface, one could still run ->fix without ->isCandidate, right? later one is just a hint "it may be worth to run it" / " no need for run it for sure"